api-docs.md

```diff
@@ -563,6 +564,42 @@ There are, however, a few catches to this:
 - Sorting is only applied at the Druid level. Therefore, the results of a sort are not guaranteed to be accurate if Fili 
 performs any post-Druid calculations on one of the metrics that you are sorting on.
 
+### TopN ###
+
+Suppose we would like to know which three pages have the top three pageview counts for each week between January and 
+September 2014. We can easily answer such a question with a `topN` query. `topN` queries allow us to ask for the top N 
+results for each time bucket in a request. Of course, a `topN` query implies that some sort of ordering has been imposed
+ on the data. Therefore, a `topN` query has two components:
+
+1. `topN=n` where `n` is the number of results to return for each bucket
+2. `sort=metricName|dir` telling Fili how to sort the results before taking the top N. See the section on 
+[sorting](#sorting) for more details about the sort clause.
+
+Going back to the sample question at the start of the section, let's see how that looks as a
+[Fili query](https://sampleapp.fili.io/v1/data/network/week/pages?metrics=pageViews&dateTime=2014-06-01/2014-08-31&topN=3&sort=pageViews|desc):
+
+    GET https://sampleapp.fili.io/v1/data/network/week/pages?metrics=pageViews&dateTime=2014-06-01/2014-08-31&topN=3&sort=pageViews|desc
+
+We want the three highest pageview counts for each week, so `n` is set to three, and the query is aggregated to the week
+granularity. Furthermore, we want the three _largest_ pagecounts. Therefore, we sort `pageViews` in descending order (the
+first entry is the highest, the second entry is the lowest, and so on).
+
+Fili also supports asking for multiple metrics in a `topN` query. Suppose we want to know the [daily average time spent (`dayAvgTimeSpent`) on the 
+three pages with the highest number of page views for each week between January 6th and September 
```

Comment:
'asking for multiple metrics' ->'supports asking for metrics in addition to the one being sorted on' 

Yes, I know that you could be sorting on more than one metric.  It would be confusing to say one(s) though.